libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java

```diff
@@ -129,4 +129,13 @@ public void telemetryInitializerMiddlewareNotLogActivitiesDisabled() {
         List<String> eventNames = eventNameCaptor.getAllValues();
         Assert.assertEquals(0, eventNames.size());
 	}
+
+	@Test
+    public void telemetryInitializerMiddlewareWithUndefinedContext() {
+        // Arrange
```

Comment:
The test has `Arrange` but missing `Act` and `Assert`

Reply:
Fixed